### PR TITLE
Added dtek.cls to english dictionary, added NC, and optional fun-mode

### PR DESCRIPTION
**Pull request by ErikLjungdahl**
Now uses the Student Division of Computer Science and Engineering's Latex-classes, to be consistent with other documents of the D-Division.

Added NC and related phrases, corrected misspelling of committee, Intresse -> Hobby, no more orphans. Also started an optional fun-mode which can be activated for a more fun reading experience.

### CODE REVIEW

File: english/dictionary.tex [hunk 1]
```diff
@@ -1,18 +1,20 @@
-\documentclass[a4paper]{article}
+\documentclass[a4paper]{dtek}
```
Comment (Hugsim) [inline english/dictionary.tex]: bra kanske

Comment (Hugsim) [inline english/dictionary.tex:1]: bra kanske, smaksak

File: english/dictionary.tex [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 
-\usepackage[english]{babel}
-\usepackage[T1]{fontenc}
-\usepackage[utf8]{inputenc}
+%\usepackage[english]{babel}
```
Comment (Hugsim) [inline english/dictionary.tex]: Borde ta bort istället

File: english/dictionary.tex [hunk 3]
```diff
@@ -23,17 +25,37 @@
 \DeclareDocumentCommand{\lit}{m}{\textit{(lit.~#1)}~}
 \DeclareDocumentCommand{\formal}{}{\textit{(formal)}~}
 
+\DeclareDocumentCommand{\fun}{m}{}
```
Comment (Hugsim) [inline english/dictionary.tex:23]: image

File: english/dictionary.tex [hunk 4]
```diff
@@ -23,17 +25,37 @@
...
+% \DeclareDocumentCommand{\fun}{}{}  % Switch to this for fun-mode
```
Comment (Hugsim) [inline english/dictionary.tex:24]: image

File: english/dictionary.tex [hunk 5]
```diff
@@ -67,12 +92,13 @@ \section{Study related terms}
 \begin{description}
     \phrase{Tenta}{Exam}[]
     \phrase{Tentamen}{Exam}[\see{Tenta}]
-    \phrase{Tentavecka}{Exam week}[\abbr{TV} A week during which exams are held. Located at the end of each study period.]
+    \phrase{Tentavecka}{Exam week}[\abbr{TV} A week during which exams are held. Located at the end of each study period. \fun{Abbreviated TV due to the procrastination that occurs during this week}]
```
Comment (Hugsim) [inline english/dictionary.tex:90]: image

File: english/dictionary.tex [hunk 6]
```diff
@@ -67,12 +92,13 @@ \section{Study related terms}
...
     \phrase{Omtentavecka}{Re-exam week}[A week during which re-exams are held. There are a couple throughout the year.]
     \phrase{Läsår}{Study year}[The academic year, running from after summer break to before summer break.]
     \phrase{Läsperiod}{Study period}[\abbr{LP} A quarter of a study year, usually consists of eight study weeks and an exam week.]
     \phrase{Läsvecka}{Study week}[\abbr{LV} A week where studying takes place, compare to for instance a \textit{Tentavecka} or a \textit{Mottagningsvecka}.]
     \phrase{Läsdag}{Study day}[A day on which studying is done. Usually non-holiday Mondays--Fridays.]
+    \fun{\phrase{Rekursion}{Recursion}[\see{Rekursion}]}
```
Comment (Hugsim) [inline english/dictionary.tex:96]: image

File: english/dictionary.tex [hunk 7]
```diff
@@ -92,24 +118,24 @@ \section{Division committees, hobby committees and organization}
 % Could do with more info about what the committees do
 \begin{description}
     \phrase{Sektionskommitté}{Division committee}[An organization under the division that is slightly more important than the division clubs.]
-    \phrase{Intressekommitté}{Hobby committee}[An often smaller and slightly more focused committee under the division.]
+    \phrase{Hobbykommitté}{Hobby committee}[An often smaller and slightly more focused committee under the division.]
```
Comment (Hugsim) [inline english/dictionary.tex:116]: Bra!

File: english/dictionary.tex [hunk 8]
```diff
@@ -92,24 +118,24 @@ \section{Division committees, hobby committees and organization}
...
     \phrase{Styret}{Board of the Division}[They decide everything between division meetings.]
     \phrase{Styrelsen}{Board of the Division}[\see{Styret}]
     \phrase{Valberedning}{Nomination committee}[]
     \phrase{DNS}{Student educational committee}[]
     \phrase{D6}{Party committee}[]
-    \phrase{Delta}{PR comittee}[]
+    \phrase{Delta}{PR committee}[]
```
Comment (Hugsim) [inline english/dictionary.tex:122]: Najs! Flytta ut i separat pull request så blir du definitely approved as they say.

**Review by Hugsim — CHANGES_REQUESTED** [review]
image

File: english/dictionary.tex (hunk 2 above)
Comment (ErikLjungdahl) [reply english/dictionary.tex]:
```suggestion
```

File: english/dictionary.tex (hunk 3 above)
Comment (ErikLjungdahl) [reply english/dictionary.tex:23]: image

File: english/dictionary.tex (hunk 4 above)
Comment (ErikLjungdahl) [reply english/dictionary.tex:24]: image

File: english/dictionary.tex (hunk 5 above)
Comment (ErikLjungdahl) [reply english/dictionary.tex:90]: image

File: english/dictionary.tex (hunk 6 above)
Comment (ErikLjungdahl) [reply english/dictionary.tex:96]: image

**Pull request closed without merging**